### PR TITLE
Annotate TODOs with links to GitHub issues

### PR DESCRIPTION
**Pull request by citizen-stig**
Initial annotation of TODOs for GitHub issues for better discoverability

### CODE REVIEW

File: sov-modules/sov-modules-api/src/lib.rs [hunk 1]
```diff
@@ -207,7 +207,5 @@ pub trait ModuleInfo {
 
     fn new() -> Self;
 
-    // Returns an address for the module.
-    // TODO: https://github.com/Sovereign-Labs/sovereign/issues/136
```
Comment (citizen-stig) [inline sov-modules/sov-modules-api/src/lib.rs:211]: @bkolad please verify that this removal is correct

File: db/sovereign-db/src/ledger_db/rpc.rs [hunk 2]
```diff
@@ -124,8 +124,8 @@ impl LedgerRpcProvider for LedgerDB {
         tx_ids: &[sovereign_sdk::rpc::TxIdentifier],
         _query_mode: QueryMode,
     ) -> Result<Vec<Option<Self::TxResponse>>, anyhow::Error> {
-        // TODO: Sort the input and use an iterator instead of querying for each slot individually
-        // https://github.com/Sovereign-Labs/sovereign/issues/191
+        // TODO: https://github.com/Sovereign-Labs/sovereign/issues/191 Sort the input
+        //      and use an iterator instead of querying for each slot individually
```
Comment (citizen-stig) [inline db/sovereign-db/src/ledger_db/rpc.rs:128]: Space is added so IDE shows second line when you search for TODOs. I hope it is not annoying

**Review by preston-evans98 — APPROVED** [review]
LGTM, pending confirmation on the last issue from @bkolad

File: sov-modules/sov-modules-api/src/lib.rs (hunk 1 above)
Comment (bkolad) [reply sov-modules/sov-modules-api/src/lib.rs:211]: Yes

**Pull request merged**